README.md

```diff
@@ -18,13 +18,11 @@ Developed by vdveen. This local web app will fetch departure information from an
 ## Installation guide:
 The app runs on NodeJS and two NPM packages. Some Raspberry Pi's might need node-semver as a dependancy for NodeJS. 
 
-`$ sudo apt-get install nodejs npm node-semver`
-
-`$ npm install express ns-api`
+`$ nam install`
```

Comment:
See down below the `package.json`. The packages are mentioned in there. When you do `npm install` it will read the json file and install all the needed libs. 

This way it is a lot easier to install and you know for sure everyone has the right version of every library, since there is also an description of which version of the library to install.